olive/passes/onnx/perf_tuning.py

```diff
@@ -332,11 +332,11 @@ def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigPa
         }
 
     def _run_for_config(self, model: ONNXModel, config: Dict[str, Any], output_model_path: str) -> ONNXModel:
-        if "providers_list" not in config:
+        if not config.get("providers_list"):
             # add the provider to the config if user doesn't provide the execution providers
             config["providers_list"] = [self._accelerator_spec.execution_provider]
```

Comment:
User should not have need to provide separate providers_list to ORTPerfTuning pass. We should remove that ability.

The ORTPerfTuning (and other places) should only get EP info from AcceleratorSpec. AcceleratorSpec should be the only place where the user can specify EP(s)

Reply:
I think the providers_list is used for standalone purpose. for example, the ORTPerfTuning could be used without engine.